library/src/main/java/com/bumptech/glide/load/engine/exceptions/ErrorWrappingGlideException.java

```diff
@@ -0,0 +1,11 @@
+package com.bumptech.glide.load.engine.exceptions;
```

Comment:
@ihenchi I don't think a new package is necessary here.

@sjudd how will we forward-port this? Use https://github.com/bumptech/glide/blob/master/library/src/main/java/com/bumptech/glide/load/engine/GlideException.java or keep this error-wrapping one?